docs/site/Debugging-with-VSC.md

```diff
@@ -0,0 +1,26 @@
+---
+lang: en
+title: 'Debugging with VS Code'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, Debug
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Debugging-with-vsc.html
+---
+
+LoopBack team develops with editor Visual Studio Code in which we defined some
+tasks to help people debug their projects.
+
+### Debugging Packages
+
+The monorepo `loopback-next` contains a folder `.vscode` at root level for
+debugging the test files. When start debugging, it will stop at any breakpoint
```

Comment:
Should we mention `.vscode/launch.json` so that it can be customized?

Reply:
👍 rephrased it to 
>There are two predefined configurations defined in file `.vscode/launch.json`: